packages/back-end/src/services/files.ts

```diff
@@ -79,9 +108,14 @@ export async function getFileUploadURL(ext: string, pathPrefix: string) {
       uploadURL,
       fileURL: S3_DOMAIN + (S3_DOMAIN.endsWith("/") ? "" : "/") + filePath,
     };
-  }
-  // Otherwise, use the local file system
-  else {
+  } else if (UPLOAD_METHOD === "google-cloud") {
+    const uploadURL = await getSignedGoogleUrl().catch(console.error);
```

Comment:
If this fails, it will just return `undefined` as the uploadURL, which could be confusing and frustrating for users since it will fail, but they won't know why. Should we just get rid of the `catch` instead and have it throw the error?

Reply:
👍